activejob/test/support/integration/test_case_helpers.rb

```diff
@@ -63,4 +63,16 @@ def job_executed_in_locale(id = @id)
     def job_executed_in_timezone(id = @id)
       job_data(id)["timezone"]
     end
+
+    def assert_job_executed(id = @id)
```

Comment:
Those are only used in one file, so why they are moved to here?

Reply:
There are two other helpers that appear to only be used in `queuing_test` but defined in `test_case_helpers`: `job_executed_in_locale` and `job_executed_in_timezone`. I don't have a strong preference for where they should go, if they should be in `queuing_test` I can update the PR.

Reply:
Yeah, let's push to the user. If there is more than 1 user we can extract.